src/library.js

```diff
@@ -2763,7 +2763,7 @@ LibraryManager.library = {
 
     if (flags & 1 /*EM_LOG_CONSOLE*/) {
       if (flags & 4 /*EM_LOG_ERROR*/) {
-        err(str);
+        console.error(str);
```

Comment:
For consistency, we should probably also update line 2774 below to use `console.log`

Reply:
```
diff --git a/src/library.js b/src/library.js
index b6a9fd245..afbb0ca34 100644
--- a/src/library.js
+++ b/src/library.js
@@ -2761,12 +2761,12 @@ LibraryManager.library = {
       } else if (flags & 256 /*EM_LOG_DEBUG*/) {
         console.debug(str);
       } else {
-        out(str);
+        console.log(str);
       }
     } else if (flags & 6 /*EM_LOG_ERROR|EM_LOG_WARN*/) {
-      err(str);
+      console.error(str);
     } else {
-      out(str);
+      console.info(str);
     }
   },
```

Does it make sense to do this ? 

from the documentation :
```
If neither :c:data:`EM_LOG_WARN`, :c:data:`EM_LOG_ERROR`, :c:data:`EM_LOG_INFO` nor :c:data:`EM_LOG_DEBUG` is specified, an info message is printed.
```

In devtools I noticed console.log and console.info are categorized in the same way, the difference must be in firefox which will add a little "i" if console.info is called

Reply:
I think `log` would make sense, especially given there is already a way to print to `info` using `EM_LOG_INFO`

Reply:
Thanks, I updated the documentation line, and forced pushed the changes